server/sonar-web/src/main/js/apps/issues/models/issue.js

```diff
@@ -0,0 +1,17 @@
+define([
+  'components/issue/models/issue'
+], function (Issue) {
+
+  return Issue.extend({
+    reset: function (attrs, options) {
+      // TODO remove me soon
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AS1135)